scripts/scil_visualize_seeds.py

```diff
@@ -40,9 +41,11 @@ def main():
         raise ValueError("Invalid input streamline file format " +
                          "(must be trk): {0}".format(args.tractogram_filename))
 
-    # Load files and data
-    trk = TrkFile.load(args.tractogram)
-    tractogram = trk.tractogram
+    # Load files and data. TRKs can have 'same' as reference
+    tractogram = load_tractogram(args.tractogram, 'same')
+    # Streamlines are saved in RASMM but seeds are saved in VOX (?)
```

Comment:
If the streamlines are saved in RASmm, maybe we should also save the seeds in RASmm? Also, if the space is non-iso, this changes the behavior of the script, right?

Reply:
> If the streamlines are saved in RASmm, maybe we should also save the seeds in RASmm?
Ideally, yes. But, dipy tracking is done is voxel space and then the tractogram is saved in RASmm. However, the data_per_streamline (or per_point, for that matter) is not modified for obvious reasons. I'm not sure how that should be handled.
> Also, if the space is non-iso, this changes the behavior of the script, right ?
I don't think so, unless the transformation from RASmm to vox doesn't handle non-iso.

Reply:
I think there is great argument for both space, and no matter which one we choose it is going to be weird for some use later.

I think the seed should be saved in vox, since in scilpy we are doing the tracking in vox space.